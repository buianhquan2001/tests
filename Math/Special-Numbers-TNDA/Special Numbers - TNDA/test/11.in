100
509317438 370176362
979877472 246408888
128386249 246862520
646154655 111554305
216024966 925213741
381474571 966989470
247168770 75246187
942445746 219454832
534062564 652769045
561561427 891376351
72047209 71658552
860928875 997572653
475506757 404347730
253962421 453551089
994265931 532272442
8429336 396019583
332982261 517577708
314318283 488572101
148141949 768950069
654051404 726904452
232101556 744180437
16860028 28873395
35962938 429492602
406731486 572755438
678871431 60832213
971532362 917011918
102922532 54922534
752006969 184379329
130442601 257501839
214550952 784060373
562233075 588187181
111480635 260222563
213811062 448143805
64307533 5091133
375556009 521253492
598134782 190590600
915962517 313387555
205548935 350670754
486303004 452573592
713068442 98551363
386060936 170461804
55395876 81401899
821746158 466251143
226993574 982440913
867646193 846547747
441387659 957700111
744036548 797654418
812520488 755884613
899712478 987240548
99836419 68625617
968740503 205409039
518269297 193555544
225073968 144573806
278225226 993261331
841117963 154112400
98580141 185110945
881806665 370330578
342909742 400378851
669129555 743581001
612998017 30036881
479240037 518791511
89578236 786368827
892638852 455203243
267542640 120673769
2991224 155533962
771428972 622551983
811548862 247516481
56658551 30332141
658796735 928516547
770723173 901459077
187688564 7818911
565419287 965092577
35296532 255441055
936335996 509679550
498456646 251545118
246252297 87307873
117375428 485751163
105236814 777276912
362010737 100647086
631398941 46729327
185996561 989078516
693981141 534465160
216367913 977801393
461062098 402191844
753390348 641939792
793647099 297842932
575230089 14283723
494349769 746341460
68794464 2146220
655422203 750297137
94786228 606807147
994615071 589771009
239596580 480270519
925172266 88015470
933234623 918516220
188838174 943219392
634902808 833907041
885386596 596701672
572968942 168544449
278661982 792004090
