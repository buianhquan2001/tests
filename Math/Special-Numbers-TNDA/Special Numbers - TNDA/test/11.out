634852745
979855442
160169855
694280835
968147525
966443922
149706569
499625130
578371627
680288421
480500292
886774712
246728942
910290537
666779555
398046031
424358162
362625921
8177561
169307701
646287313
693208775
298621380
470804856
430168991
983564658
56100994
202023136
109278673
357328155
190109783
883752754
893184989
202162541
252211851
681056599
478376313
797193553
842817233
852060551
753090217
85516409
65245067
553013002
646248356
552045396
364049936
104709822
333191329
425211917
155788576
123973512
103932018
527579495
453877486
482502368
880560170
797125141
867832985
752502587
144987860
915513229
447059035
891176433
642855088
392564785
222306673
668536729
814394527
178035107
768136827
671405164
666031442
346639675
343141810
78576924
659492362
858164212
606487492
189399971
815179187
200850535
987721013
576697650
505851549
521981590
289911796
193915397
227387188
912351206
181293660
741623253
910196405
122842350
248810080
775837646
526058014
543078159
966706588
381215425
